journals/apps/journals/models.py

```diff
@@ -401,6 +407,16 @@ def root_journal_page_url(self):
         descendants = self.get_descendants()
         return descendants[0].full_url if descendants else '#'
 
+    def structure(self):
```

Comment:
If we're renaming from an action `get_...`, we should probably make it a `@property` instead. I had thought you could do something like `APIField('get_journal_structure', name='structure')`, but that seems to not be the case.

Reply:
you can use a serializer, but it maybe overkill for what we need: http://docs.wagtail.io/en/v2.1.1/advanced_topics/api/v2/configuration.html#custom-serialisers